app/views/content/train-to-be-a-teacher/promos/_events-near-you.html.erb

```diff
@@ -0,0 +1,22 @@
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      caption: "24 May 2022",
+      heading: "Virtual Train to Teach event",
+      link_target: "/events/train-to-teach-national-virtual-event-240522",
+      link_text: "Sign up for this event") do %>
+    <p>Attend an exclusive panel session made up of expert advisers, teachers and training providers.</p>
```

Comment:
@EllieNodder Think we should probably try and highlight that it is not just a one-way presentation?

Eg something like 'Hear from training providers, chat with teachers and get one-to-one advice from expert advisers.'

(but have a look at the event details and feel free to change it!)